addons/event/data/event_data.xml

```diff
@@ -9,7 +9,7 @@
         </record>
         <record id="event_stage_booked" model="event.stage">
             <field name="name">Booked</field>
-            <field name="description">The place has been reserved</field>
+            <field name="description"></field> <!-- Necessary to clear previous value during the upgrade -->
```

Comment:
I would phrase it in a way that it's not confused with a simple `update` of a module.

```suggestion
            <field name="description"></field> <!-- Necessary to set a void value to clear the previous value when module `event` is upgraded to 17.4 -->
```